src/IVSPlayer.tsx

```diff
@@ -239,7 +231,7 @@ const IVSPlayerContainer = React.forwardRef<IVSPlayerRef, Props>(
         duration: number | null;
       }>
     ) => {
-      if (!paused) {
+      if (autoplay) {
         play();
       } else {
```

Comment:
With this change the behavior would now become:

**With autoplay = true**
`play -> load is called, load event is raised -> play`

**With autoplay = false**
`play -> load is called, load event is raised -> pause`
During the `load is called, load event is raised` step it is possible for the video to begin playing, then pause.